### DISCUSSION TITLE
It looks like all of the discussions in github got cleared out??

### DISCUSSION THREAD

**Discussion by jordancrubin** [deta/space-docs#16] [General]
I guess Ill start.   Seems to be a button only for the addition of a collection, but none for the deletion of one.  Any reason for this?

**Comment by sudo-subg** [comment]
@jordancrubin we are currently resolving a few minor deployment issues to ensure a smooth deployment process. The option to delete a Collection, and many other features are planned. Hopefully soon 🙂

**Comment by abdelhai** [comment]
the old discussions are here btw: https://github.com/deta/cloud-docs/discussions
awesome youtube channel! are you using base for any of your hardware projects?
